### PR TITLE
Faqaddition

### LINKED ISSUE #69: Add FAQ questions.
These questions came in over the weekend:

“I still cannot understand, what is relation between Aspire and K8S. Will Aspire just yet another way to deploy to K8S (like, fancy helm with c# instead of yaml)? Also, is Aspire dashboard created for demos, or you want to replace Grafana with it?”

### PR DESCRIPTION
**Pull request by alistairmatthews**
## Summary

Add a FAQ describing the relationship between Aspire and Kubernetes.

Fixes #69

### CODE REVIEW

File: docs/deployment/azure/includes/aca-deployment-cli.md
Comment (IEvangelist) [inline docs/deployment/azure/includes/aca-deployment-cli.md:1]: Undo these changes

**Comment by IEvangelist** [comment]
Hey, @alistairmatthews - this seems to include part of another PR that was already approved/merged. Can you remove the changes to the CLI include?

**Comment by alistairmatthews** [comment]
Hi @IEvangelist, my bad - branched off the wrong branch. I'll sort it out and re-create. Thanks!
